### DISCUSSION TITLE
R Project: "No Symbol" Problem Reported

### DISCUSSION THREAD

**Discussion by StatsResearch** [posit-dev/positron#5717] [Q&A] [answered]
Hi Folks

Not sure if this is a bug or I could fix with a setting.

If I open an R project I see a number of "problems" reported
with "No symbol xxx" See (NoSymbol_01.pdf)

If I then run the lines 1 to 8, the library() calls. Positron
is now happy and says there are no problems.

NoSymbol_02.pdf shows the here library ran OK. 
 
NoSymbol_03.pdf shows Positron reports No Problems. 

Not a problem in such a small file but in a real file

I can open the project and be faced with well over 50 problems.

Let me know if I can provide any more information.

Bye

Rob

I've tried Positron versions 2024.12.0 (Universal) build 96
and Positron Version: 2025.01.0 (Universal) build 39
(see file PositronVersions.txt)

[NoSymbol_01.pdf](https://github.com/user-attachments/files/18115013/NoSymbol_01.pdf)
[NoSymbol_02.pdf](https://github.com/user-attachments/files/18115014/NoSymbol_02.pdf)
[NoSymbol_03.pdf](https://github.com/user-attachments/files/18115015/NoSymbol_03.pdf)
[PositronVersions.txt](https://github.com/user-attachments/files/18115016/PositronVersions.txt)

**Comment by jennybc** [comment, marked as answer]
Thanks for test driving Positron!

We are tracking this in #1325. For R packages specifically, there's also #2252. You're welcome to give either of those a 👍 (or add details if you think your findings aren't already captured.)

**Reply by StatsResearch** [reply]
Thanks for the quick reply. Sorry I didn't spot the earlier mentions. Looks like a difficult one.
